ktor-server/ktor-server-plugins/ktor-server-request-validation/jvmAndNix/src/io/ktor/server/plugins/requestvalidation/RequestValidation.kt

```diff
@@ -72,6 +76,19 @@ public val RequestValidation: RouteScopedPlugin<RequestValidationConfig> = creat
             throw RequestValidationException(content, failures.flatMap { it.reasons })
         }
     }
+
+    if (!pluginConfig.validateContentLength) return@createRouteScopedPlugin
+
+    on(ReceiveRequestBytes) { call, body ->
+        val contentLength = call.request.contentLength() ?: return@on body
+
+        return@on application.writer {
```

Comment:
should it be scope of a call?

Reply:
ApplicationCall doesn't have a coroutine scope